salt/metalk8s/orchestrate/solutions/prepare-environment.sls

```diff
@@ -155,8 +154,8 @@ Cannot deploy Solution {{ name }}-{{ version }} for environment {{ env_name }}:
                                  | selectattr('version', 'equalto', version)
                                  | first %}
 
-              {{- deploy_operator(namespace, solution) }}
-              {{- deploy_admin_ui(namespace, solution) }}
+              {{- deploy_operator(namespace, name, solution) }}
+              {{- deploy_admin_ui(namespace, name, solution) }}
```

Comment:
Maybe calling this macro should be made optional based on the presence of a `spec.ui.image` key in the manifest? Or we can just ignore until #2581 is done